### ISSUE TITLE
use ANT+ Radar (RDR) instead of heart-rate

### ISSUE THREAD

**Issue by mfa** [tobst/radmesserS#14]
https://www.thisisant.com/news/ant-wireless-releases-first-interoperable-device-profile-for-bike-radar-sen

**Comment by tobst** [comment]
Good Idea. Since not all devices (e.g. IPhone) support ANT+, this should be an additional choice.
